views/about.tmpl

```diff
@@ -0,0 +1,40 @@
+{{define "content"}}
+<div class="container-fluid current-next">
+  <div class="container">
+    <div class="row align-items-center">
+      <div class="col">
+        <h1 class="text-center">About Us</h1>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="container-fluid container-padded bg-off-white">
+	<div class="container">
+    <h1 class="user-title">Welcome to URY!</h1>
+    <div class="row">
+      <div class="col-md-6 col-lg-3">
+        <h2>Who are URY?</h2>
+        <p>University Radio York (URY) is the student radio station for the University of York, broadcasting 24 hours a day during term time on 1350am across the University's Heslington Campus and worldwide via this website.</p>
```

Comment:
Well, it would mean a tonne load of custom CSS instead of using bootstrap's built in functions. I'd rather be able to see what the page should do (and have the support of bootstrap documentation) then some custom stuff.